.github/CODEOWNERS

```diff
@@ -1111,3 +1111,4 @@
 
 #PRLabel: %CodeGen %Client
 /eng/packages/http-client-csharp/generator/Azure.Generator/        @JoshLove-msft @m-nash @jorgerangel-msft @jsquire @annelo-msft
+/eng/scripts/typespec/                                             @JoshLove-msft @m-nash @jorgerangel-msft @jsquire @annelo-msft
```

Comment:
This will assign owners but has no associated labels.   A label only applies to the block that directly follows.   Because this is a new path, it's a new block.  I think what you want is:

```text
#PRLabel: %CodeGen %Client
/eng/packages/http-client-csharp/generator/Azure.Generator/        @JoshLove-msft @m-nash @jorgerangel-msft @jsquire

#PRLabel: %CodeGen %Client
/eng/scripts/typespec/                                             @JoshLove-msft @m-nash @jorgerangel-msft @jsquire
```